lib/ssl/src/tls_dtls_connection.erl

```diff
@@ -832,21 +829,18 @@ override_server_random(Random, _, _) ->
     Random.
 
 new_server_hello(#server_hello{cipher_suite = CipherSuite,
-			      compression_method = Compression,
-			      session_id = SessionId},
-                 #state{session = Session0,
-                        static_env = #static_env{protocol_cb = Connection}} = State0, Connection) ->
+                               session_id = SessionId},
+                 #state{session = Session0} = State0, Connection) ->
```

Comment:
do you really want to remove pattern matching on protocol_cb. when it's removed different function clause might be selected ... just speculating. I don't know if it was done on purpose ...

here and below ...